packages/desktop-client/src/components/Notifications.tsx

```diff
@@ -185,7 +185,7 @@ function Notification({
                   backgroundColor: positive
                     ? theme.noticeBackground
                     : error
-                    ? theme.altErrorBackground
+                    ? theme.errorBackground
```

Comment:
<img width="572" alt="Screenshot 2023-10-07 at 14 21 22" src="https://github.com/actualbudget/actual/assets/886567/74633d83-d4ea-4ee1-9224-5d5eff992673">

nit: I think a white background here would be nicer to add more contrast.

Reply:
I think any material changes to the color patterns should be a separate PR. This PR is just to consolidate error color text/border/backgrounds, it's not meant to be used for color or contrast changes.

Reply:
What do you mean? You're already doing real changes to the colors here. It's not a no-op update PR. Arguably the previous color is closer to white than red (but this is borderline).

| Before | After |
|--------|--------|
| <img width="590" alt="Screenshot 2023-10-10 at 20 21 17" src="https://github.com/actualbudget/actual/assets/886567/f1991bd3-c9e3-4db5-8bcb-0041a48201bc"> | <img width="601" alt="Screenshot 2023-10-10 at 20 21 03" src="https://github.com/actualbudget/actual/assets/886567/26175665-6ce1-4e5d-9e4a-4f71a01dcd16">  |

Reply:
So you want it white for error but green for notices and yellow for warnings? Shouldn't all these be consistant. Like if we change one to white shouldn't they all be white? If not, if only the error is meant to be white, shouldn't we then change that for all other errors where we use the red background? 

This is my point, we are creating inconsistency to how we deal with colors. This PR isn't concerned with consistency across all pages, that is a much bigger PR. I'm just trying to create consistency with the colors themselves - like when we use a red background for error text they should all use the same red background. I'm trying to take this in smaller chunks so as to not create some massive PR that is impossible to review.